include/cpuinfo.h

```diff
@@ -2042,6 +2043,18 @@ static inline bool cpuinfo_has_arm_sve2(void) {
 #endif
 }
 
+// Function to get the max SVE vector length on ARM CPU's which support SVE.
+static inline int cpuinfo_get_max_arm_sve_length(void) {
+#if CPUINFO_ARCH_ARM64
+	if ((cpuinfo_isa.sve || cpuinfo_isa.sve2) && (cpuinfo_isa.svelen > 0))
+		return cpuinfo_isa.svelen;
```

Comment:
If I understood your previous diff correctly, svelen is returned in bytes, but API developers usually think about it in bitlength, therefore it must be multiplied by 8, shouldn't it?
```suggestion
		return cpuinfo_isa.svelen * 8;
```

Reply:
Yes its better to return bit length directly compared to bytes to avoid confusion to the end users.
Changes done.